File: app/src/main/java/app/country/information/model/Idd.java

package app.country.information.model;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class Idd {

    @SerializedName("root")
    @Expose
    public String root;
    @SerializedName("suffixes")
    @Expose
    public List<String> suffixes = null;

}

File: app/src/main/java/app/country/information/model/Idd.kt
package app.country.information.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

class Idd {
    @SerializedName("root")
    @Expose
    var root: String? = null

    @SerializedName("suffixes")
    @Expose
    var suffixes: List<String>? = null
}